plugins/rtmp-services/dacast.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+struct dacast_ingest {
+	const char *url;
+	const char *url2;
+	const char *username;
+	const char *password;
+	const char *streamkey;
+
+	const char *video_codec;
+	const char *h264_profile;
+	double frame_rate;
+	int keyframe_interval;
+	const char *scanning;
+	const char *rate_control;
+	const char *audio_codec;
+	int audio_bitrate;
+	int audio_channels;
+	int audio_sample_rate;
+};
```

Comment:
There are a lot of fields here that seem unused.

Reply:
As commented below, we'll remove this and discuss settings with you separately.